contracts/test/TeleporterTokenDestinationTests.t.sol

```diff
@@ -102,6 +119,23 @@ abstract contract TeleporterTokenDestinationTest is TeleporterTokenBridgeTest {
         );
     }
 
+    function _encodeMessage(
+        SendTokensInput memory input,
+        uint256 amount
+    ) internal virtual override returns (bytes memory) {
```

Comment:
This uses `_scaleTokens()`, which can technically modify state, so this function couldn't be pure anymore.